### PR TITLE
FRW-5858 Added description how to API specification file download in data exchange API.

### PR DESCRIPTION
**Pull request by geega**
Ticket: https://spryker.atlassian.net/browse/FRW-5858

### CODE REVIEW

File: docs/scos/dev/glue-api-guides/202307.0/data-exchange-api/how-to-guides/how-to-configure-data-exchange-api.md [hunk 1]
```diff
@@ -130,3 +130,20 @@ If everything is set up correctly, you can follow [How to send request in Data E
 Or if you're in the middle of the integration process for the Data Exchange API follow [Data Exchange API integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/glue-api/data-exchange-api-integration.html) to proceed with it.
 
 {% endinfo_block %}
+
+## Download Data Exchange API specification
+
+You can download the specification of the Data Exchange API endpoints in the OpenAPI format. To do so, click **Download Data Exchange API specification** on the **Data Exchange API Configuration** page.
+
+[PASTE SCREENSHOT HERE]
```
Comment (geega) [inline docs/scos/dev/glue-api-guides/202307.0/data-exchange-api/how-to-guides/how-to-configure-data-exchange-api.md:138]: download-btn

File: docs/scos/dev/glue-api-guides/202307.0/data-exchange-api/how-to-guides/how-to-configure-data-exchange-api.md [hunk 2]
```diff
@@ -130,3 +130,20 @@ If everything is set up correctly, you can follow [How to send request in Data E
...
+After you click the button, the specification will be downloaded to your computer.
+
+If you added new endpoints, documentation will be updated automatically with the new endpoints, but need wait 1 minute for it. You can see  message about it in the top of the page:
+
+[PASTE SCREENSHOT HERE]
+
+So. please wait 1 minute and refresh the page. After that you will see new endpoints in the documentation.
+Also if for some reason file not exists, you will see the message about it:
+
+[PASTE SCREENSHOT HERE]
```
Comment (geega) [inline docs/scos/dev/glue-api-guides/202307.0/data-exchange-api/how-to-guides/how-to-configure-data-exchange-api.md:149]: Screenshot 2023-09-06 at 15 13 03

File: docs/scos/dev/glue-api-guides/202307.0/data-exchange-api/how-to-guides/how-to-configure-data-exchange-api.md [hunk 3]
```diff
@@ -130,3 +130,20 @@ If everything is set up correctly, you can follow [How to send request in Data E
...
+
+You can download the specification of the Data Exchange API endpoints in the OpenAPI format. To do so, click **Download Data Exchange API specification** on the **Data Exchange API Configuration** page.
+
+[PASTE SCREENSHOT HERE]
+
+After you click the button, the specification will be downloaded to your computer.
+
+If you added new endpoints, documentation will be updated automatically with the new endpoints, but need wait 1 minute for it. You can see  message about it in the top of the page:
+
+[PASTE SCREENSHOT HERE]
```
Comment (geega) [inline docs/scos/dev/glue-api-guides/202307.0/data-exchange-api/how-to-guides/how-to-configure-data-exchange-api.md:144]: Screenshot 2023-09-06 at 15 17 41

File: docs/scos/dev/glue-api-guides/202307.0/data-exchange-api/how-to-guides/how-to-configure-data-exchange-api.md (hunk 2 above)
Comment (geega) [inline docs/scos/dev/glue-api-guides/202307.0/data-exchange-api/how-to-guides/how-to-configure-data-exchange-api.md:149]: Screenshot 2023-09-06 at 15 10 51

**Review by asmarovydlo — CHANGES_REQUESTED** [review]
1. Please mention somewhere that documentation generation might be in progress, and the button will appear whenever the documentation generation is finished.
2. Why the screenshots added just as a comment? Will TW upload the images?

**Comment by geega** [comment]
@asmarovydlo 

1. Added text about button. 
2. Yes. I don't know where to download the pictures. I see that this is an s3 bucket, but I don't have access to it. So I did like on PR for basic branch

**Comment by geega** [comment]
@asmarovydlo  
I added screenshots `docs/scos/dev/drafts-dev`
@lenadoc helped me with it. 
https://spryker.atlassian.net/wiki/spaces/DOCS/pages/3554246794/Contributing+to+docs+for+Spryker+developers


> Currently, only technical writers can add images to documentation. If you need to insert an image to your doc, add it [here](https://github.com/spryker/spryker-docs/tree/master/docs/scos/dev/drafts-dev). The technical writer who will review your PR will move the image to the proper storage.

File: notes.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
+# Notes
```
Comment (asmarovydlo) [inline notes.md:1]: Where did those file changes come from? They are not related to the current documentation

File: docs/scos/dev/glue-api-guides/202307.0/data-exchange-api/how-to-guides/how-to-configure-data-exchange-api.md [hunk 5]
```diff
@@ -130,3 +130,25 @@ If everything is set up correctly, you can follow [How to send request in Data E
...
+
+***Also if for some reason file not exists, you will see the message about it***
+
+[PASTE SCREENSHOT deapi_spec_file_not_exists.png HERE]
+
+Please regenerate specification in this case manually or add new endpoints for Data Exchange API.
+
+```bash 
+ vendor/bin/glue api:generate:documentation
+```
```
Comment (asmarovydlo) [inline docs/scos/dev/glue-api-guides/202307.0/data-exchange-api/how-to-guides/how-to-configure-data-exchange-api.md:154]: This is no longer the case, this functionality is removed. Please align with Dmitry Tsemma, and remove those lines and screenshot.

**Review by asmarovydlo — CHANGES_REQUESTED** [review]

**Review by asmarovydlo — DISMISSED** [review]

**Review by asmarovydlo — APPROVED** [review]

**Comment by asmarovydlo** [comment]
> The base branch was changed.
It is OK, @lenadoc could you please make TW review, and if all is good - merge?

**Pull request merged**